frontend/src/components/DisplayIcon.tsx

```diff
@@ -18,7 +16,7 @@ export default function DisplayIcon({
     <div
       data-tooltip-id={`icon-tooltip-${item}`}
       data-tooltip-content={`${Icons[item as keyof typeof Icons]?.label}`}
-      className={`flex flex-col items-center justify-center gap-1 border-b border-l border-border px-4 pb-1 ${idx == 0 ? 'rounded-bl-md' : ''}`}
+      className={`flex flex-col items-center justify-center gap-1 border border-border px-4 pb-1 ${idx == 0 ? 'rounded-bl-md' : ''}`}
```

Comment:
I understand that you added this to accomodate the mobile view to have the border, but we didn't put a top border here, so that there's no double border on the desktop view. Right now, it's pretty noticeable when we view it in desktop mode:
![Screenshot 2024-12-29 at 2 44 36 PM](https://github.com/user-attachments/assets/28dbf764-c206-47f2-802e-60c4afa5610a)
![Screenshot 2024-12-29 at 2 44 54 PM](https://github.com/user-attachments/assets/0564db63-5eab-43cb-ae3a-e0f741083178)

Also, as you can see there was a bottom left corner radius added. This was also only intended for mobile view to match with a top right corner of the card. 
If we just move this block down under the title - it does look weird with that corner being rounded 🤔 

My original idea was to show these as blocks with borders only for desktop view, and then show it somehow differently (as a list as an option) for mobile view. That might be too much to address within this PR since I would I like to merge it in at least for the footer changes. 

If you could just update this to only show rounded  bottom-left corner and top-border for mobile devices - this should work for now! We can address the rest in a different task.
What do you think?